server/app/auth/oidc/OidcProvider.java

```diff
@@ -179,7 +179,9 @@ public OidcClient get() {
     Optional<String> providerName = getProviderName();
     OidcConfiguration config = new OidcConfiguration();
 
+    config.setLogoutUrl("https://civiform-staging.us.auth0.com/v2/logout");
```

Comment:
This seems like something that was missed. If I understand correctly, we don't want to reference specific URLs from server implementation.

Reply:
Blocking: This still appears to be present.

Reply:
It's fixed now